.github/workflows/CompatHelper.yml

```diff
@@ -0,0 +1,16 @@
+name: CompatHelper
+on:
+  schedule:
+    - cron: 0 0 * * *
+  workflow_dispatch:
+jobs:
+  CompatHelper:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Pkg.add("CompatHelper")
```

Comment:
Isn't it better to collapse these two steps into one so Julia only has to be started one time?

Reply:
It is like this at https://github.com/JuliaRegistries/CompatHelper.jl, so I guess Dilum Aluthge has good reasons for it. My guess is for readability reasons.

Reply:
By the way, I generated this file from PkgTemplates. As a fun project, I'm planning to run a weekly CI job to compare the generated PkgTemplates files with Example.jl.